compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -1470,7 +1470,10 @@ object messages {
     extends Message(CannotExtendAnyValID) {
     val msg = hl"""$sym cannot extend ${"AnyVal"}"""
     val kind = "Syntax"
-    val explanation = ""
+    val explanation =
+      hl"""Only classes (not traits) are allowed to extend ${"AnyVal"}, but traits may extend
+          |${"Any"} to become ${Green("\"universal traits\"")} which may only have ${"def"} members.
```

Comment:
I might be good to also mention that classes extending `AnyVal` can extend `universal traits`.